pages/user guides/discover_nmap.md

```diff
@@ -8,58 +8,53 @@ permalink: discover-nmap.html
 ---
 
 
+As part of your cloud migration journey, it is important to have all the tools at your disposal. To facilitate the discovery process, you can use [Nmap](https://nmap.org/) in addition to [the other discovery methods](https://guides.tidalmg.com).
 
-As part of your cloud migration journey it is important to have all the tools at your disposal. In addition to [the other discovery methods](https://guides.tidalmg.com), you can use [Nmap](https://nmap.org/) to facilitate a part of the discovery process. 
-The way it works is very simple and with the full power of Nmap available to you, discovery of even those hard to reach places is possible. 
- 
-Nmap has developed some powerful capabilities and good [documentation](https://nmap.org/book/host-discovery-find-ips.html) that will allow you to identify all the hosts you have running on your network, the ports and services that they have open and running. 
- 
-Nmap is capable of producing its output in an XML file, allowing you to inspect the raw scan output before sending it to the Tidal Migrations API with [Tidal Tools](https://get.tidal.sh). Once uploaded to the Tidal Migrations Platform, you will be able to visualize your network devices, track your complete server inventory and build on this data with other discovery methods, allowing you to make informed decisions on your cloud migration path.
+With Nmap's powerful capabilities and good [documentation](https://nmap.org/book/host-discovery-find-ips.html), you can identify all the hosts you have running on your network, open ports, and running services. It is possible to discover those places which are often hard to reach.
+
+Nmap is capable of producing its output in an XML file. It allows you to inspect the raw scan output before sending it to the Tidal Migrations API with [Tidal Tools](https://get.tidal.sh). Once uploaded to the Tidal Migrations Platform, you will be able to visualize your network devices, track your complete server inventory, and build on this data with other discovery methods. This is how you can make informed decisions on your cloud migration path.
 
 
 ## Using Nmap with Tidal Tools
- 
-By leveraging the power of **Tidal Tools** you can send the output generated by Nmap to your Tidal Migrations account.
- 
+By leveraging the power of **Tidal Tools**, you can send the output generated by Nmap to your Tidal Migrations account.
 1. Install [Tidal Tools](https://guides.tidalmg.com/tidal-tools.html)
-2. [Login to Tidal Tools](https://guides.tidalmg.com/tidal-tools.html#login) with **tidal login**
-3. Run Nmap ([Need to install Nmap?](#installing-nmap)) with the flags of your choosing and save the output to an XML file. For example, 
+2. [Login to Tidal Tools](https://guides.tidalmg.com/tidal-tools.html#login) with **Tidal login**
```

Comment:
“Login to Tidal Tools” makes no sense — it actually logs in to Tidal Migrations account. So let's change it into something like the following:

**Connect Tidal Tools and your Tidal Migrations account with `tidal login`.**

Reply:
mmm, Yeah. I see your point 👍.  It did make sense to me until now. 
Nice suggestion!